README.md

```diff
@@ -21,9 +25,12 @@ Sketch.App Patch Tool, brought to you by [@duraki](https://github.com/duraki) &
 ## Magic Trick ✨
 For the people who would like to try the new version, we got you covered. You can pass `-m` argument for the ultimate life-saving trick, which will automagically download latest Sketch app from the official website and patch the bundle, ready to be launched from the Applications folder.
 
-And do it in one command: `bash -c "$(curl -s https://raw.githubusercontent.com/duraki/SketchCrapp/master/sketchcrapp.sh -o -)" -O -m`
+[@gordio](https://github.com/gordio) added<br />And do it in one command: `bash -c "$(curl -s https://raw.githubusercontent.com/duraki/SketchCrapp/master/sketchcrapp.sh -o -)" -O -m` 
```

Comment:
Rewrite as:
---


One-liner script to install latest Sketch version and automatically patch it:

```
bash -c "$(curl -s https://raw.githubusercontent.com/duraki/SketchCrapp/master/sketchcrapp.sh -o -)" -O -m
```